openssl-sys/src/handwritten/ssl.rs

```diff
@@ -679,6 +679,10 @@ cfg_if! {
             pub fn TLS_server_method() -> *const SSL_METHOD;
 
             pub fn TLS_client_method() -> *const SSL_METHOD;
+
+            // DTLS 1.2 support doesn't exist in LibresSSL 2.9.1
+            #[cfg(ossl110)]
+            pub fn DTLSv1_2_method() -> *const SSL_METHOD;
```

Comment:
Given that this symbol's cfg doesn't align with the rest of the cfg_if block, I'd just move it out to the non-cfg-if block below

Reply:
Done - plese let me know if that's acceptable.